JS/Lesson24/saffet/map.js

```diff
@@ -179,10 +183,16 @@ const studentsExercise5 = [
 	},
 ];
 
+const studentAndGrade = studentsExercise5.map(student => {
```

Comment:
This does no actually extracts the array of values that you need but only prints the mentioned values one by one to console
```suggestion
const studentAndGrade = studentsExercise5.map(student => student.name + " " + student.grade);
console.log(studentAndGrade);
```